core/src/main/java/com/loop54/model/request/GetPopularOrRecentEntitiesRequest.java

```diff
@@ -0,0 +1,38 @@
+package com.loop54.model.request;
+
+import java.util.List;
+
+import com.loop54.model.request.parameters.EntityCollectionParameters;
+
+/// <summary>Base class for requests to get the most popular or most recent entities.</summary>
+public abstract class GetPopularOrRecentEntitiesRequest extends Request
+{
+    public static final String CURRENT_USER_PLACEHOLDER = "(CurrentUser)";
```

Comment:
Would be nice to have a comment on this constant, same as in the .NET connector.